.github/workflows/deploy.yml

```
@@ -53,4 +53,9 @@ jobs:
           npx lerna publish from-package \
             --force-publish='*' \
             --yes
-      - uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # v0.1.15
+      - run: |
+          PKG_VERSION=$(jq -r .version < lerna.json)
+          gh release create $PKG_VERSION \
+            --title "Release $PKG_VERSION" \
+            --generate-notes \
+            --repo ${{ github.repository }}
```

This will use the default branch `develop` to create the release from. Probably want to use `master`

```suggestion
          gh release create $PKG_VERSION \
            --title "Release $PKG_VERSION" \
            --generate-notes \
            --repo ${{ github.repository }}
            --target master
```